freemocap/core_processes/export_data/blender_stuff/export_to_blender/methods/legacy/run_megascript_take2.py

```diff
@@ -52,17 +52,17 @@ def run_blender_megascript_take2(
         str(blender_file_path),
     ]
 
-    logger.info(f"Starting `blender` sub-process with this command: \n {command_list}")
+    logger.debug(f"Starting `blender` sub-process with this command: \n {command_list}")
```

Comment:
see above (let me know if "above" makes sense in the medium you wind up reading this)